server/src/test/java/io/druid/server/coordinator/DruidCoordinatorTest.java

```diff
@@ -130,6 +132,9 @@ public ScheduledExecutorService create(int corePoolSize, final String nameFormat
       }
     };
     leaderAnnouncerLatch = new CountDownLatch(1);
+
+    metadataNamespaceManager = EasyMock.createNiceMock(MetadataNamespaceManager.class);
+    EasyMock.replay(metadataNamespaceManager);
```

Comment:
why do when need a reply ? i don't see any mock expectation to replay and verify or i am missing something ?